src/CarbonAware.Aggregators/src/Configuration/ServiceCollectionExtensions.cs

```diff
@@ -14,7 +16,8 @@ public static class ServiceCollectionExtensions
     public static IServiceCollection AddCarbonAwareEmissionServices(this IServiceCollection services, IConfiguration configuration)
     {
         services.AddDataSourceService(configuration);
-        services.TryAddSingleton<ICarbonAwareAggregator, CarbonAwareAggregator>();
+        services.TryAddSingleton<IForecastAggregator, ForecastAggregator>();
+        services.TryAddSingleton<IEmissionsAggregator, EmissionsAggregator>();
```

Comment:
Is the thought we'd always add both even if we only intend to use one? (I imagine that would be a good case for the null interface)

Reply:
For now, yes.  Longterm it would be great to actually turn off endpoints at the Consumer Tier based on config and avoid the need for the null object pattern in the core SDK altogether.

EG
* The CLI would only register the `emissions-forecasts` command if a forecast data source was configured;
* The API only includes an `EmissionsController`, and thus the corresponding API endpoint, if the emissions data source is configured.